### DISCUSSION TITLE
Deploy Edgedb database to the cloud

### DISCUSSION THREAD

**Discussion by haikyuu** [geldata/gel#2873] [Show and tell]
I made an open source app with Edgedb, node.js, Inertia and Imba and I'd like to deploy it to the cloud. What's the easiest way to do it. Where can I find a guide?

https://github.com/haikyuu/perfect-stack

Thank you

**Comment by 1st1** [comment]
This is awesome! @elprans is working on a deployment guide, we'll update you on it shortly

**Reply by haikyuu** [reply]
Sure, thanks a lot. 
I'm also excited about the query builder. It looks amazing !!
I'll migrate to it as soon as it's ready 💖

**Reply by jamilabreu** [reply]
Similarly looking for a cloud deployment guide 👋 
Are there informal docs somewhere? If so happy to contribute

**Reply by jamilabreu** [reply]
Specifically on top of Fly would be amazing! https://twitter.com/edgedatabase/status/1450703809722785798?s=20

Again even informal docs would be appreciated!
